next/components/atoms/Breadcrumbs.tsx

```diff
@@ -101,31 +106,33 @@ const Breadcrumbs = ({ crumbs, className }: BreadcrumbsProps) => {
         {isExpanded || !isCollapsing ? (
           <div className="flex items-center gap-1 py-6">{breadcrumbedChildren}</div>
         ) : (
-          <div className="flex w-full flex-col">
-            <div className="flex w-full items-center justify-between">
-              <div className="flex items-center gap-1">
+          <div className="flex w-full flex-col py-6">
+            <div className="flex w-full items-center justify-between overflow-hidden">
+              {/* min-w-0 is used because:
+              https://css-tricks.com/flexbox-truncated-text/#aa-the-solution-is-min-width-0-on-the-flex-child */}
+              <div className="flex min-w-0 items-center gap-1">
                 {/* first child */}
                 {breadcrumbedChildren[0]}
                 {/* dots, linkHref is ignored */}
-                <BreadcrumbChild crumb={{ label: '…', path: '#' }} noLink />
+                <BreadcrumbChild crumb={{ label: '…', path: '#' }} noLink dontShrink />
                 {/* last child */}
                 {last(breadcrumbedChildren)}
               </div>
               {/* TODO add aria attributes */}
-              <button className="p-4" type="button" onClick={() => setOpen((o) => !o)}>
+              <button className="px-4" type="button" onClick={() => setOpen((o) => !o)}>
```

Comment:
Padding on icon buttons is used to expand the clickable/touchable area, mainly on touch screens. Consider if this does not shrink the clickable area of the button. :)

Reply:
Fixed.